crates/rust-analyzer/src/cli/load_cargo.rs

```diff
@@ -15,12 +15,13 @@ use crate::reload::{ProjectFolders, SourceRootConfig};
 
 pub fn load_cargo(
     root: &Path,
+    config: &CargoConfig,
     load_out_dirs_from_check: bool,
     with_proc_macro: bool,
```

Comment:
Let's extract these three into `struct LoadCargoConfig` while we still have a managable number of parameters.

Reply:
:+1: